<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="slideDiffuse">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slideBlackout">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="openBlackout">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #E6C548; /* Choose your desired background color */
    color: white; /* Choose your desired text color */
    border: 1px solid #E6C548; /* Add a border for better visibility */
    border-radius: 20px;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBlackout">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #E6C548; /* Choose your desired background color */
    color: white; /* Choose your desired text color */
    border: 1px solid #E6C548; /* Add a border for better visibility */
    border-radius: 20px;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openDiffuse">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #E6C548; /* Choose your desired background color */
    color: white; /* Choose your desired text color */
    border: 1px solid #E6C548; /* Add a border for better visibility */
    border-radius: 20px;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeDiffuse">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>210</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #E6C548; /* Choose your desired background color */
    color: white; /* Choose your desired text color */
    border: 1px solid #E6C548; /* Add a border for better visibility */
    border-radius: 20px;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Blackout</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Blackout</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Diffuse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Diffuse</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
